frontend/src/metabase/writeback/containers/CategoryRadioPicker.tsx

```diff
@@ -49,7 +49,7 @@ function CategoryRadioPicker({
 
   useOnMount(() => {
     if (fieldInstance.id) {
-      fetchFieldValues({ id: fieldInstance.id });
+      fetchFieldValues({ id: fieldInstance.getId() });
```

Comment:
FYI @kulyk  -- this function is typed to expect a `number` id, but `Fields` are awful classes, and the `id` can actually be an entire field_ref